frontend/src/components/LandingPage/TeamCard/TeamPage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import TeamCard from './TeamCard';
+import { Container, Row, Col } from 'reactstrap';
+
+
```

Comment:
A lot of the `<Col></Col>` are similar; the only difference between them are the data being passed to `<TeamCard />`. Because of this, you can make an array above the function that holds all the data. 

Then inside `<Row>` and `</Row>`, replace everything with the code below. The code below is equivalent to all the `<Col></Col>` that were manually typed in. 
```
{
    members.map((member, index) => (
        <Col key={index} md="4" sm="12">
            <TeamCard {...member} />
        </Col>
    ))
}
```
Refer to [here](https://reactjs.org/docs/lists-and-keys.html) for more information.